src/index.js

```diff
@@ -1,17 +1,34 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import React, {lazy, Suspense} from 'react';
+// import ReactDOM from 'react-dom';
+// import './index.css';
+// import App from './App';
+// import reportWebVitals from './reportWebVitals';
 
-ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-  document.getElementById('root')
-);
+import {Route, Routes} from 'react-router-dom';
 
-// If you want to start measuring performance in your app, pass a function
-// to log results (for example: reportWebVitals(console.log))
-// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals();
+const Home = lazy(()=>import('./routes/Home/Home'));
+const About = lazy(()=>import('./routes/About/About'));
+
+export default function AppRoutes() {
+  return (
+    <Suspense fallback={<p>Loading</p>}>
+      <Routes>
+        <Route path={'/'} element={<Home />} />
+        <Route path={'/about'} element={<About />} />
+      </Routes>
+    </Suspense>
+  );
+}
+
+
+// ReactDOM.render(
+//   <React.StrictMode>
+//     <App />
```

Comment:
Ok, so the key issue I can see here is you have the purpose / contents of the app and index files reversed. Here the index.js loads, but the bootstrapping of React (`reactDOM.render()`) never happens, so you have a blank screen.

The Index is the root for the JS application, and that loads the App. 

Check the `** src` section of the email for more details.

So, set index back to how it was with this section uncommented, and move the routing code above into to App.js and you should get something rendering, or at least some console messages calling out any errors.

Aside: in the Annie example in that last email you'll see the routes themselves are further abstracted into a <Routes /> component, which is exported from the `routes/index.js` (screenshotted). It would be fine to have them in App.js, but there will eventually be a bunch of stuff in App JS, so abstracting the routes into a separate, dedicated file helps keep things easy to read.